src/documents/tests/test_management_exporter.py

```diff
@@ -497,3 +506,80 @@ def test_export_target_not_writable(self):
                 call_command(*args)
 
                 self.assertEqual("That path doesn't appear to be writable", str(e))
+
+    def test_no_archive(self):
```

Comment:
It's great to include testing.  Please use the [given/when/then](https://en.wikipedia.org/wiki/Given-When-Then) style commenting to help quickly show what the test is doing and the expected output

Reply:
fixed by c0466755d8b3cc0ae5dbed6993b4392d83f210c6